README.md

```diff
@@ -1,12 +1,24 @@
 # xblock-group-project-v2
 
-
 This XBLock is experimental reimplementation of [Group Project XBlock](https://github.com/edx-solutions/xblock-group-project).
 
 It does *not* work properly in LMS yet.
 
 ## Features
```

Comment:
It would be nice to have a short high-level summary of what this xblock is for before going into a detailed list of features.

This is my first look at the group work xblock - I assume it's something to do with working in groups given the name, but it's not very clear. The link to the other xblock doesn't help much either, as the README for that project is empty.